hc/accounts/views.py

```diff
@@ -208,6 +208,7 @@ def login(request: HttpRequest) -> HttpResponse:
         "registration_open": settings.REGISTRATION_OPEN,
         "support_email": settings.SUPPORT_EMAIL,
         "account_closed": "account-closed" in request.GET,
+        "magic_link_enabled": settings.EMAIL_HOST not in (None, ""),
```

Comment:
My naming preference, used in various places in this project, is `use_<somefeature>`. In this case, I suggest `"use_magic_form": ...`.

`settings.EMAIL_HOST not in (None, "")` will miss another falsy value, `False`. I think we should test for a falsy value like this instead: `bool(settings.EMAIL_HOST)`.